.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,3 +1,5 @@
+<!-- Love storybook? Please consider supporting our collective: 👉  https://opencollective.com/storybook/donate -->
```

Comment:
Pushed to the bottom. This is just a suggestion, feel free to change the text or to ignore this PR.
Just thought this was worth a shot to get feedback from the community (in my experience this is typically the kind of thing that a lot of people would have an opinion about, but once it's done, no one cares because it really doesn't bother anyone). Worst case, it's also easily reversible :)

Up to you!